src/tests/unit/tests/background/global-context-factory.test.ts

```diff
@@ -20,6 +21,7 @@ import { CreateTestAssessmentProvider } from '../../common/test-assessment-provi
 
 describe('GlobalContextFactoryTest', () => {
     let _mockChromeAdapter: IMock<BrowserAdapter>;
+    let _mockStorageAdapter: IMock<StorageAdapter>;
```

Comment:
nit: do you want to get rid of the leading `_` ?

Reply:
not as part of this PR... keeping the in-file style consistent though